homeassistant/components/dwd_weather_warnings/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "To identify the desired region, either the warncell ID / name or device tracker is required. The provided device tracker has to contain the attributes 'latitude' and 'longitude'.",
+        "data": {
+          "name": "Name of the sensor",
+          "region_identifier": "Warncell ID or name",
+          "gps_tracker": "Device tracker entity",
+          "scan_interval": "Scan interval",
```

Comment:
We don't want configurable scan intervals. Just set a sensible default.

Reply:
Ahh forgot to remove that one, the option didn't even exist anymore